addons/account/views/account_move_views.xml

```diff
@@ -342,6 +342,7 @@
                     <filter string="Non Trade Payable" domain="[('account_id.account_type', '=', 'liability_payable'), ('account_id.non_trade', '=', True)]" help="From Non Trade Receivable accounts" name="non_trade_payable" invisible="1"/>
                     <filter string="Non Trade Receivable" domain="[('account_id.account_type', '=', 'asset_receivable'), ('account_id.non_trade', '=', True)]" help="From Non Trade Receivable accounts" name="non_trade_receivable" invisible="1"/>
                     <filter string="P&amp;L Accounts" domain="[('account_id.internal_group', 'in', ('income', 'expense'))]" help="From P&amp;L accounts" name="pl_accounts"/>
+                    <filter string="No Statement Line" domain="[('statement_line_id', '=', False)]" name="no_st_line_id" invisible="1"/>
```

Comment:
Uh.
Does it really need to be a filter? Can't it be a custom one for people who need it? I don't think it's a common domain.

Reply:
Invisible